app/Commands/ReposCommand.php

```
@@ -0,0 +1,413 @@
+    }
+
+    private function viewPullRequests($repo): void
+    {
+        info("🔀 Launching pull requests for {$repo->full_name}...");
+
+        // Call the prs command with the specific repo
+        $this->call('prs', [
+            '--repo' => $repo->full_name,
+            '--format' => 'interactive',
+        ]);
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'viewPullRequests'. (Unused Code Rules)

(UnusedPrivateMethod)